app/controllers/application_controller.rb

```diff
@@ -44,8 +44,12 @@ def routing_error
       when Common::Exceptions::BaseError
         exception
       when Common::Client::Errors::BackendServiceError
-        additional_attributes = { source: exception.developer_message, meta: exception.as_json }
-        Common::Exceptions::ClientError.new(exception.message, additional_attributes)
+        additional_attributes = {
+          detail: exception.message,
+          source: exception.developer_message,
+          meta: exception.as_json
+        }
+        Common::Exceptions::BackendServiceError.new(nil, additional_attributes)
```

Comment:
Haha, sounds like someone is going to be confused one way or the other on this one. @carlmjohnson this change was mostly meant to address concerns @aub brought up on the client_error not being clear enough -- I'm hoping that eventually one or the other of these will go away entirely, but I can't think of a good way to facilitate this yet. Maybe once I've had to revisit the one on client side in Phase 3 of this refactor.